lib/ansible/modules/system/gather_facts.py

```diff
@@ -85,6 +85,7 @@
       their output must be formattable in JSON (Ansible will take care of this). Test the
       output of your scripts.
       This option was added in Ansible 2.1.
+    - the module used to be named 'setup', changed in version 2.4.
```

Comment:
Capitalize the first letter of this sentence. Use `I(setup)` instead of `'setup'`.